scene/gui/tree.cpp

```diff
@@ -2177,7 +2177,7 @@ int Tree::draw_item(const Point2i &p_pos, const Point2 &p_draw_ofs, const Size2
 				}
 			}
 
-			if (!rtl && p_item->cells[i].buttons.size()) {
+			if (p_item->cells[i].buttons.size()) {
```

Comment:
```suggestion
			if (!p_item->cells[i].buttons.is_empty()) {
```
Could as well fix the style if you are changing this line.